src/components/Modal/Modal.jsx

```diff
@@ -199,7 +199,8 @@ class Modal extends React.PureComponent {
 
     if (focusedNodeIndex === 0) {
       event.preventDefault()
-      focusableNodes[focusableNodes.length - 1].focus()
+      const i = focusableNodes.length - 1
```

Comment:
should we defend for the possibility that focusableNodes.length could be 0 which make this try to access index of -1?

Reply:
yes good catch, I did fixed it in 2d6cabbf